### ISSUE TITLE
"startReturnTimer" not called in "widget_pagebutton.js"

### ISSUE THREAD

**Issue by juniormajor** [knowthelist/fhem-tablet-ui#169]
since last update the return function within the pagebutton isn't triggered anymore.
last known working version 23b63a961a (pre "Improved pointer events handler")

**Comment by knowthelist** [comment]
can't trace this. no issue found

**Comment by juniormajor** [comment]
To clarifiy the issue:
until the rework of the pointer events, pagebutton has the same capability
as pagetab regarding the return to a specified page after x seconds ()
defined as attr return-time.
Since the rework the return timer isn't started at all. So automatic return
to a specific page isn't possible anymore.

Or do you mean you need logs?

Am 24.10.2016 2:12 vorm. schrieb "knowthelist" notifications@github.com:

**Comment by knowthelist** [comment]
The condition to trigger the return timer is:
data-return-time of the first pagebutton >0 and at least tap/click of the screen after an activation of a new pagebutton.

```
       // start return timer after last activity
       if ( me.elements.eq(0).data('return-time') > 0 ){
           $('body').once('touchend mouseup', function(e) {
               startReturnTimer(me.elements.eq(0));
               e.preventDefault();
           });
       }
```

If you just change the view via pagebutton and don't click on the screen, the timer wont start. Maybe this is the problem, isn't it? Should be changed to: return in any case.

**Comment by juniormajor** [comment]
indeed, absolutely correct! nevertheless old behaviour was that the timer was triggered without "additional click". just change the view via click onto the pagebutton and timer was triggered.

would it be hard to reimplement the old behaviour?

2016-10-26 6:32 GMT+02:00 knowthelist notifications@github.com:
